documentation/src/docs/metrics.md

```diff
@@ -116,204 +101,158 @@ public final class ServiceStarter {
-                                                       final long createTimestamp,
-                                                       final OperationContext context,
-                                                       final Executor executor) {
-            return withCompletion(getStream(scope, name, context).create(configuration, createTimestamp), executor)
-                    .thenApply(result -> {
-                        if (result.getStatus().equals(CreateStreamResponse.CreateStatus.NEW)) {
-                            CREATE_STREAM.reportSuccessValue(1); // Report success event for histogram metric CREATE_STREAM
-                            dynamicLogger.reportGaugeValue(nameFromStream(OPEN_TRANSACTIONS, scope, name), 0); // Report gauge value for Dynamic Gauge metric OPEN_TRANSACTIONS 
-                        }
-    
-                        return result;
-                    });
-        }
+    public void createStream(String scope, String streamName, int minNumSegments, Duration latency) {
+        DYNAMIC_LOGGER.reportGaugeValue(OPEN_TRANSACTIONS, 0, streamTags(scope, streamName));
+    }
     ...
  }
 ```
 
 ### 2.3 Example for Dynamic Meter
-This is an example from io.pravega.segmentstore.server.SegmentStoreMetrics. In this class, we report a Dynamic Meter which represents the segments created.
+This is an example from io.pravega.segmentstore.server.SegmentStoreMetrics. In this class, we report a Dynamic Meter 
+which represents the segments created with a particular container.
 ```java
 public final class SegmentStoreMetrics {
-    private final DynamicLogger dynamicLogger = MetricsProvider.getDynamicLogger();
-    
-    public void createSegment() {
-            dynamicLogger.recordMeterEvents(this.createSegmentCount, 1);  // Record event for meter metric createSegmentCount
-    } 
+    private static final DynamicLogger DYNAMIC_LOGGER = MetricsProvider.getDynamicLogger();
+
+    public final static class Container {
+        private final String[] containerTag;
+
+        public Container(int containerId) {
+            this.containerTag = containerTag(containerId);
+        }
+
+        public void createSegment() {
+            DYNAMIC_LOGGER.recordMeterEvents(MetricsNames.CONTAINER_CREATE_SEGMENT_COUNT, 1, containerTag);
+        }
+    }
 }
 ```
  
-# 3. Metric reporter and Configurations
-Reporters are the way through which we export all the measurements being made by the metrics. We currently provide StatsD and CSV output. It is not difficult to add new output formats, such as JMX/SLF4J.
-CSV reporter will export each Metric out into one file. 
-StatsD reporter will export Metrics through UDP/TCP to a StatsD server.
+# 3. Metric Registries and Configurations
+With Micrometer, each meter registry is responsible for both storage and exporting of metrics objects. 
+In order to have a unified interface Micrometer provides CompositeMeterRegistry for application to interact with,
```

Comment:
Consider to use `this format` with the class names (`CompositeMeterRegistry`). We are trying to enforce a consistent style in the docs progressively.

Reply:
Accepted suggestion